### PR TITLE
Draft of PyJulia recipe.

### PR DESCRIPTION
**Pull request by Carreau**
As discussed in https://github.com/JuliaPy/pyjulia/issues/126.
So far only on Linux, as Julia itself in linux-only on conda-forge.

The package name differs on PyPI where it is "just" julia (but we can
rename the pypi version if needed).

cc @ViralBShah 

I've put you and me as maintainer, let me know who else to add. 

Let see if this builds.

### CODE REVIEW

File: recipes/pyjulia/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - julia
+
+test:
+  commands:
+    - python -c 'import julia; julia.Julia()'
+
+about:
+  home: https://github.com/JuliaPy/pyjulia
+  license: MIT
+  license_file: LICENSE
```
Comment (jakirkham) [inline recipes/pyjulia/meta.yaml]: Please change to `license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'`.

Also please add a comment above this line referencing PR ( https://github.com/JuliaPy/pyjulia/pull/137 ). We can switch to getting it from source once that is merged and released.

File: recipes/pyjulia/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  license_family: MIT
+  summary: "python interface to julia"
+
+extra:
+  recipe-maintainers:
+    - Carreau
+    - ViralBShah
```
Comment (jakirkham) [inline recipes/pyjulia/meta.yaml]: Are you ok being listed as a maintainer on this recipe, @ViralBShah?

File: recipes/pyjulia/meta.yaml (hunk 1 above)
Comment (jakirkham) [reply recipes/pyjulia/meta.yaml]: Should add that doing this should fix the current CI failure.

File: recipes/pyjulia/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{% set version = "0.1.4" %}
+{% set name = "julia" %}
+
+package:
+  name: pyjulia
```
Comment (jakirkham) [inline recipes/pyjulia/meta.yaml]: **Note to others:** The package is called `julia` on PyPI. Though that will conflict with the `julia` package (compiler/interpreter) in `conda-forge`, so using the repo name is a reasonable second choice. Hence it is called `pyjulia` here.

**Comment by jakirkham** [comment]
This seems fine to me once the issues above are addressed.

File: recipes/pyjulia/meta.yaml (hunk 2 above)
Comment (ViralBShah) [reply recipes/pyjulia/meta.yaml]: I doubt I really can contribute in any meaningful way directly, but will be happy to help in any way I can.

Comment (jakirkham) [reply recipes/pyjulia/meta.yaml]: There's honestly not too much required from individual maintainers. Here is a [description]( https://github.com/conda-forge/staged-recipes#9-what-does-being-a-conda-forge-feedstock-maintainer-entail ).

File: recipes/pyjulia/meta.yaml (hunk 3 above)
Comment (Carreau) [reply recipes/pyjulia/meta.yaml]: > This seems fine to me once the issues above are addressed.

We're going to get a new version with the LISENCE.md file released on PyPI, and them I'll update that. 

@ViralBShah we can also rename on PyPI to pyjulia for consistency, thoughts ?

Comment (ViralBShah) [reply recipes/pyjulia/meta.yaml]: Yes,  I think that makes sense for consistency.

**Comment by Carreau** [comment]
Seem to build now, but the `import julia` test seem to fail with `No module named julia`, not sure why.  Let's see if this fails only on Circle,

**Comment by jakirkham** [comment]
Added `--ignore-installed` to the `pip` command. Expect that fixes the install issue, which the CIs seem to confirm. However this runs into another issue. Presumably because `PyCall` is not installed.

```python
ArgumentError("Module PyCall not found in current path.\nRun `Pkg.add(\"PyCall\")` to install the PyCall package.")Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/conda/staged-recipes/build_artifacts/pyjulia_1525057450635/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python2.7/site-packages/julia/core.py", line 370, in __init__
    self._call(u"using PyCall")
  File "/home/conda/staged-recipes/build_artifacts/pyjulia_1525057450635/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python2.7/site-packages/julia/core.py", line 412, in _call
    self.check_exception(src)
  File "/home/conda/staged-recipes/build_artifacts/pyjulia_1525057450635/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python2.7/site-packages/julia/core.py", line 432, in check_exception
    .format(exception_type, src))
julia.core.JuliaError: Exception 'ArgumentError' occurred while calling julia code:
using PyCall
```

**Pull request closed without merging**